src/test/java/test/tesng1046/TestCustomNamesForTests.java

```diff
@@ -0,0 +1,33 @@
+package test.tesng1046;
```

Comment:
For name tests, we already have https://github.com/cbeust/testng/tree/master/src/test/java/test/name
Yours could be `test.name.github1046`

Fyi: testngXXXX was the key for the previous bug tracker.